### ISSUE TITLE
Cleaning Code

### ISSUE THREAD

**Issue by naveen521kk** [ManimCommunity/ManimPango#30]
There is a significant copy-pasting of the same code, for `Text` and `MarkupText`, instead if needs to be cleaned up so it is more maintainable.

**Comment by PhilippImhof** [comment]
I suggest having some sort of `AbstractText` class containing all common parts (where it is sensible) and then have `Text` and `MarkupText` as extensions of `AbstractClass`.

**Comment by naveen521kk** [comment]
@PhilippImhof are you interested in doing this?

**Comment by PhilippImhof** [comment]
Yes, absolutely. I will draft a concept and discuss it with you before I start coding. But I need some time...
